docs/vm-images.md

```diff
@@ -166,45 +206,63 @@ a description of our support for Windows images.
 
 <sup><b>*</b></sup> Chrome Remote desktop does not support Ubuntu 20.04, but it does support Ubuntu 22.04.
 
-## Other Images
+### Other Images
 
 The HPC Toolkit strives to provide flexibility wherever possible. It is possible
 to set a VM image in many HPC Toolkit modules. While we do not officially
 support images not listed here, other public and custom images should work with
 the majority of modules with or without further customization, such as custom
 startup-scripts.
 
-## Slurm on GCP Custom Images
+## Slurm on GCP
+
+### Publicly Published Slurm Images
+
+SchedMD publishes "Slurm on GCP" public images, which are documented
+[here][slurm-gcp-images]. This documentation covers which images are available
+and what software is installed on them.
+
+> **_Note:_** When deploying a Slurm cluster, the HPC Toolkit Terraform wrapper
+modules enforce that specific image versions are used, to ensure compatibility.
+Currently, the **slurm-gcp-5-7** image families are enforced for the Slurm v5
+modules provided in the HPC Toolkit.
+
+For Slurm images, please see
+[SchedMD's GitHub repository](https://github.com/SchedMD/slurm-gcp/blob/5.7.6/docs/images.md#public-image).
+
+### Custom Slurm Images
+
+> **_Warning:_** When building custom images, the Terraform and Packer modules
+must share the same version.
 
 HPC Toolkit modules based on terraform modules in [Slurm on GCP][slurm-gcp]
```

Comment:
I would reduce the rest of the text to provide 2 links to `examples/README.md` with the anchors for image-builder.yaml and ml-slurm.yaml.

Reply:
is this resolved?

Reply:
Yes, resolved.